Pinta/Actions/View/ImageTabsToggledAction.cs

```diff
@@ -47,7 +47,7 @@ public void Uninitialize ()
 		private void Activated (bool value)
 		{
 			// TODO-GTK3 (docking)
```

Comment:
Since this is being re-enabled, could you please remove the TODO comment and `#if` block?